### PR TITLE
Add LiquidationPremiumMultiplier parameter

### PR DESCRIPTION
**Pull request by leomassazza**

### CODE REVIEW

File: contracts/PerpsV2MarketData.sol [hunk 1]
```diff
@@ -119,15 +119,15 @@ contract PerpsV2MarketData {
             );
     }
 
-    function _futuresMarketSettings() internal view returns (IPerpsV2MarketSettings) {
+    function _perpsV2MarketSettings() internal view returns (IPerpsV2MarketSettings) {
```
Comment (leomassazza) [inline contracts/PerpsV2MarketData.sol:122]: fix old naming

File: contracts/MixinPerpsV2MarketSettings.sol [hunk 2]
```diff
@@ -34,17 +34,18 @@ contract MixinPerpsV2MarketSettings is MixinResolver {
...
     bytes32 internal constant PARAMETER_OFFCHAIN_PRICE_DIVERGENCE = "offchainPriceDivergence";
+    bytes32 internal constant PARAMETER_LIQUIDATION_PREMIUM_MULTIPLIER = "liquidationPremiumMultiplier";
 
     // Global settings
     // minimum liquidation fee payable to liquidator
-    bytes32 internal constant SETTING_MIN_KEEPER_FEE = "futuresMinKeeperFee";
+    bytes32 internal constant SETTING_MIN_KEEPER_FEE = "perpsV2MinKeeperFee";
     // maximum liquidation fee payable to liquidator
-    bytes32 internal constant SETTING_MAX_KEEPER_FEE = "futuresMaxKeeperFee";
+    bytes32 internal constant SETTING_MAX_KEEPER_FEE = "perpsV2MaxKeeperFee";
```
Comment (davidvuong) [inline contracts/MixinPerpsV2MarketSettings.sol:43]: Nice. I hope the deploy process won't be too difficult.

File: contracts/PerpsV2MarketSettings.sol [hunk 3]
```diff
@@ -354,6 +362,10 @@ contract PerpsV2MarketSettings is Owned, MixinPerpsV2MarketSettings, IPerpsV2Mar
         _setParameter(_marketKey, PARAMETER_OFFCHAIN_PRICE_DIVERGENCE, _offchainPriceDivergence);
     }
 
+    function setLiquidationPremiumMultiplier(bytes32 _marketKey, uint _liquidationPremiumMultiplier) public onlyOwner {
```
Comment (davidvuong) [inline contracts/PerpsV2MarketSettings.sol:365]: Should we revert if _liquidationPremiumMultiplier is 0?

**Review by davidvuong — APPROVED** [review]
Left one non-blocking comment but this LGTM.

File: contracts/PerpsV2MarketSettings.sol (hunk 3 above)
Comment (leomassazza) [reply contracts/PerpsV2MarketSettings.sol:365]: good call

**Pull request merged**